src/KafkaFlow/Consumers/IConsumerWorker.cs

```diff
@@ -10,7 +10,7 @@ internal interface IConsumerWorker : IWorker, IDisposable
 
         IDependencyResolver WorkerDependencyResolver { get; }
 
-        ValueTask EnqueueAsync(IMessageContext context, CancellationToken stopCancellationToken);
```

Comment:
Why are you removing the cancellation token? It is a good practice to have a way to cancel the async execution.

Reply:
This change is related to the fix, the enqueuing should not be canceled when stopping and the message should complete its lifetime (buffered, processed, committed)

Reply:
But without the cancellation token it can lead to a block if the processing take a long time or it may cause problems due to abrupt terminations, right?

Reply:
In version 3, it is necessary to call the Complete() method for all messages to release the StopAsync() call. When the Complete() method is called, a new slot is created in the buffer, freeing up the EnqueueAsync() call and allowing this last message to complete its lifecycle.